app/src/main/java/com/kks/nimblesurveyjetpackcompose/ui/presentation/home/HomeScreen.kt

```diff
@@ -288,28 +330,36 @@ fun BottomView(
                         width = Dimension.fillToConstraints
                     }
             )
-            Button(
-                onClick = { navigator.navigate(SurveyDetailScreenDestination(survey = survey)) },
-                modifier = Modifier
-                    .clip(CircleShape)
-                    .size(56.dp)
-                    .constrainAs(detailBtn) {
-                        end.linkTo(parent.end, 20.dp)
-                        top.linkTo(parent.top)
-                        bottom.linkTo(parent.bottom)
-                    },
-                colors = ButtonDefaults.buttonColors(backgroundColor = Color.White)
-            ) {
-                Image(
-                    painter = painterResource(id = R.drawable.ic_baseline_arrow_forward_ios_24),
-                    contentDescription = stringResource(id = R.string.home_survey_detail_button)
-                )
-            }
+            SurveyRoundedButton(
+                modifier = Modifier.constrainAs(detailBtn) {
+                    end.linkTo(parent.end, 20.dp)
+                    top.linkTo(parent.top)
+                    bottom.linkTo(parent.bottom)
+                },
+                onClick = onClickSurveyDetail
+            )
             Spacer(modifier = Modifier.width(20.dp))
         }
     }
 }
 
+@Composable
+fun SurveyRoundedButton(modifier: Modifier = Modifier, onClick: () -> Unit) {
```

Comment:
Do we need this composable? Why not just use a `FloatingActionButton` instead? 🤔